### DISCUSSION TITLE
New Ideas.

### DISCUSSION THREAD

**Discussion by firecerne** [UkrainianBanderasCat/Digital-Logic-Sim2#61] [Polls]
Hey !
Someone on the original itch.io page suggested some cool stuff, and while I think it would be very hard to implement 
and would probably feel out of place, it gave me some ideas .

He also suggested something which could be very interesting which is a survival and crafting RPG based on Digital Logic Sim 2 (if you want more infos you can go on his post [here](https://sebastian.itch.io/digital-logic-sim/comments)).
While it would be very hard to make, it gave me another idea. It would be very fun for the player to be able to control a little robot on a little adventure around a city where people ask him to help them by making complex circuits (I had an idea of a military man wanting an encryption and decryption chip with very strict requirements so it will be easier to test if the player actually made it like required).
You can give me feedback on this idea in the discussion instead of in the poll.

**Comment by AlphaCodingPilot** [comment]
Maybe when you start a new project you can select a goal like a processor, an ALU or just free build and it would give you all the requirements for that chip via a checklist.

**Comment by UkrainianBanderasCat** [comment]
Or even you can earn coins and after that buy chips like RAM, ALU , so you don't need to do them

**Reply by firecerne** [reply]
Good idea, but maybe it should let you buy cosmetics (themes like you suggested before) instead of chips because, if the tasks are hard enough, the player already know how to do "simple" chips like that. Or they could be able to buy more annoying to build chips like really big XOR (for example 16bit XOR) which would work like normal XOR (not like odd number detector).
